tests/test_calendar.py

```
@@ -24,335 +24,331 @@
-        self.assertEqual(ymd_from_day(-152385, proleptic=False), (1582, 10,  4))
-
-        self.assertTrue(np.all(np.array(ymd_from_day([-152384,-152385], proleptic=True))
-                               == ([1582,1582],[10,10],[15,14])))
-        self.assertTrue(np.all(np.array(ymd_from_day([-152384,-152385], proleptic=False))
-                               == ([1582,1582],[10,10],[15, 4])))
-
-        # There's some weirdness surrounding March 1, 1 BCE (= astronomical year 0)
-        for proleptic in (True, False):
-            day = np.arange(day_from_ymd(-101, 1, 1, proleptic=proleptic),
-                            day_from_ymd(-102, 1, 2, proleptic=proleptic))
+def test_calendar():
```

Ruff (0.15.6)

[warning] Too many statements (211 > 50)

(PLR0915)